src/components/List.js

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { usePages } from '../hooks/usePages';
+import Page from './Page';
 
-const List = () => <h1>List</h1>;
+const List = () => {
+  const centerMap = {
+    // lat: 0,
+    // lng: 0,
+    lat: 48.8566,
+    lng: 2.3522,
+  };
+  const { pages, loading } = usePages(centerMap);
 
+  return (
+    <div>
+      <h1>List</h1>
+      <ul className="pages">
+        {loading && <p>Loading... </p>}
```

Comment:
the `<p>` tag is not allowed nested within a `<ul>` tag. one approach for fixing this would be to move your conditional outside the list and have it control whether the loading indicator or the entire list is rendered.

Reply:
Thanks! we moved this loading part outside `ul`, and added a `Spinner` component.

![image](https://user-images.githubusercontent.com/8689897/126830103-56cc3cf7-7df9-42bb-abb8-b25138059727.png)